src/Routing/Route/Route.php

```diff
@@ -184,11 +184,18 @@ public function setMethods(array $methods)
     /**
      * Set regexp patterns for routing parameters
      *
+     * If any of your patterns contain multibyte values, the `multibytePattern`
+     * mode will be enabled.
+     *
      * @param array $patterns The patterns to apply to routing elements
      * @return $this
      */
     public function setPatterns(array $patterns)
```

Comment:
Since other methods like setPass(), setPersist() which also take array argument are singular, wouldn't it be better to name this method too setPattern() for consistency?